scss/components/_search-bar.scss

```diff
@@ -205,6 +205,10 @@ $search-button-width: u(5.6rem);
   border-bottom: 1px solid $base;
 }
 
+.tt-highlight {
+  background-color: rgba($aqua, .3);
```

Comment:
To clarify I'm wondering if there should be a leading zero on a decimal point or if a shortened style is preferred.